stripe/resource.py

```diff
@@ -936,6 +946,11 @@ def transfers(self, **params):
         return transfers
 
 
+# This resource can only be instantiated when expanded on a BalanceTransaction
+class RecipientTransfer():
```

Comment:
Looking through this again, I think the main thing you need is a `construct_from` implementation, so you might be okay just inheriting from `StripeObject`. We should at least run a manual test to make sure this one works though when the API is available.